plugins/woocommerce-admin/client/marketplace/components/product-card/product-card.scss

```diff
@@ -0,0 +1,111 @@
+@import '../../stylesheets/_variables.scss';
+
+.woocommerce-marketplace {
+	&__product-card {
+		padding: $large-gap;
+		border-radius: $grid-unit-05 !important;
+		margin-top: $large-gap;
+
+		&__content {
+			display: grid;
+			grid-template-columns: 1fr;
+			align-items: flex-start;
+			gap: $medium-gap;
+			justify-content: space-between;
+		}
+
+		&__header {
+			grid-row: auto;
+			grid-column: 1;
+			align-self: stretch;
+		}
+
+		&__icon {
+			width: $grid-unit-60;
+			height: $grid-unit-60;
+			flex-shrink: 0;
+			border-radius: $grid-unit;
+		}
+
+		&__details {
+			display: flex;
+			justify-content: flex-start;
+			align-items: center;
```

Comment:
To prevent the situation where icons don't line up:

<img width="1576" alt="image" src="https://github.com/woocommerce/woocommerce/assets/53293/94e0d17d-8297-4b20-a2e4-1289be90eeb8">

How about: 

```suggestion
			align-items: flex-start;
```